tests/database/odbc-with-cksp/enclave-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "args": [
+        "odbc_app",
```

Comment:
@prp Any reason why this works at all? Shouldn't only `./odbc_app` and `/odbc_app` work here?

Reply:
Maybe because I have set `WORKDIR` in Dockerfile?

Reply:
I followed this sample([https://github.com/lsds/sgx-lkl/blob/oe_port/samples/basic/helloworld/enclave_config.json](https://github.com/lsds/sgx-lkl/blob/oe_port/samples/basic/helloworld/enclave_config.json)) to set the config file. As you can see, it is not necessary to put `./` or `absolute path` before the binary

Reply:
The binary is accessed with a regular `open` call, so the path will be interpreted relative to the current working directory.